media-libs/opusfile/opusfile-0.10.ebuild

```diff
@@ -24,6 +24,8 @@ DEPEND="${RDEPEND}
 
 REQUIRED_USE="^^ ( fixed-point float )"
 
+PATCHES=( "${FILESDIR}/http-fix-compilation-with-LibreSSL.patch" )
```

Comment:
Generally we patch just the newest version.

Reply:
Thanks, didn't know that. I'll resend the patch without patching 0.10 then.